src/components/cards/cards.comp.tsx

```diff
@@ -39,7 +39,7 @@ export const Cards = ({ items, fields, callbacks, customActions, customLabels, p
     }
 
     if (value && typeof value === 'object') {
-      return 'object';
+      return value.toString();
```

Comment:
Did you mean `JSON.stringify()`?

Reply:
These appear because I branched from master but targeted the dev branch with my PR. I should have been consistent. Should I target master instead?

Reply:
You should pull from `master` anyway since we're going to merge your branch there.

Reply:
But looking at `master`, it looks ok. Merging.